docs/src/starknet/account-import-export.md

```diff
@@ -0,0 +1,146 @@
+# Importing And Exporting Accounts
+
+You can export your account (more precisely its private key) from wallet (Argent, Braavos) and import it into the file holding the accounts info (`~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default).
+
+## Exporting an Account
```

Comment:
I'd name it something along the lines of `Getting Your Private Key` or similar.

Reply:
`Exporting Your Private Key` should be fine 👍